frontend/src/metabase/query_builder/components/view/ViewHeader/components/HeaderBreadcrumbs/HeaderBreadcrumbs.jsx

```diff
@@ -40,22 +53,35 @@ export function HeadBreadcrumbs({
             {isValidElement(part) ? (
               part
             ) : (
-              <HeaderBadge
+              <Badge
+                className={cx(HeaderBreadcrumbsS.HeaderBadge, {
```

Comment:
:x: Font size is off in collection name:

![image](https://github.com/user-attachments/assets/34ac9bbd-89be-4368-9fe8-0b2817033883)

:x: and also in question title when there are filters

![image](https://github.com/user-attachments/assets/cad8ec66-10ee-40f6-b93d-a98a819e92be)

Reply:
Both should be fixed

<img width="570" alt="image" src="https://github.com/user-attachments/assets/10a517dc-51d1-430c-af67-7cb99954a58c">

<img width="916" alt="image" src="https://github.com/user-attachments/assets/9714334e-0ea2-49de-a3d7-d0b9f300f03a">